types/bayes-classifier/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "allowSyntheticDefaultImports": true,
```

Comment:
older code bases will not have this set by default, so you should remove it.

Reply:
I removed `allowSyntheticDefaultImports`, but left `"lib": ["es6"]` because I would get errors if it was removed.  Was this what you wanted?

Reply:
Yep!